idea/src/org/jetbrains/kotlin/idea/quickfix/RemoveDefaultParameterValueFix.kt

```diff
@@ -0,0 +1,27 @@
+package org.jetbrains.kotlin.idea.quickfix
+
+import com.intellij.openapi.editor.Editor
+import com.intellij.openapi.project.Project
+import org.jetbrains.kotlin.diagnostics.Diagnostic
+import org.jetbrains.kotlin.psi.KtFile
+import org.jetbrains.kotlin.psi.KtParameter
+
+class RemoveDefaultParameterValueFix(parameter: KtParameter) : KotlinQuickFixAction<KtParameter>(parameter) {
+
+    override fun getText() = "Remove default parameter value"
+
+    override fun getFamilyName() = text
+
+    override fun invoke(project: Project, editor: Editor?, file: KtFile) {
+        val parameter = element ?: return
+        val typeReference = parameter.typeReference ?: return
+        val defaultValue = parameter.defaultValue ?: return
+        parameter.deleteChildRange(typeReference.nextSibling, defaultValue)
```

Comment:
What if we have a comment between type reference and `= defaultValue`, like this: `a: Int /* very important */ = 42`?